src/Keypad.h

```diff
@@ -99,6 +99,13 @@ class Keypad : public Key {
 	bool keyStateChanged();
 	byte numKeys();
 
+// Testing methods
+	bool keysScanned() { return _scanned; }
```

Comment:
Functions that return booleans are often named with a state of being (is/has/did). With this name I first thought that it was returning a collection of keys (as in "Which keys were scanned?"). I see that you test this for true. When would it test for false? Does it return useful information that isn't available from the original library's public API? Similar questions for `keyFound()` and `stateChanged`()`.